docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx

```diff
@@ -22,7 +24,14 @@ import filter_contract from "/img/hubble/filter-contract.png";
 import persistent_only from "/img/hubble/persistent-only.png";
 import filtered_report from "/img/hubble/filtered-report.png";
 
-This page will guide you through an end-to-end analysis use case with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+This page will show how you can create visualizations with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en).
+
+By the end of the tutorial you will have created two graphs that will help visualize temporary VS persistent contract data entry distribution and expiration.
+
+<img src={hubble_visualization_demo} width="1000" />
```

Comment:
Love the gif, it really shows the end user how they can customize their visuals. 
wdyt about dropping the `hubble_visualization_final`? it's kinda weird to me to include both at the top, feels duplicative

Reply:
Hmm yeah I'll drop it